engine/swarm/secrets.md

```diff
@@ -374,24 +376,46 @@ generate the site key and certificate, name the files `site.key` and
     > This example does not require a custom image. It puts the `site.conf`
     > into place and runs the container all in one step.
 
-    ```bash
-    $ docker service create \
-         --name nginx \
-         --secret site.key \
-         --secret site.crt \
-         --secret site.conf \
-         --publish 3000:443 \
-         nginx:latest \
-         sh -c "ln -s /run/secrets/site.conf /etc/nginx/conf.d/site.conf && exec nginx -g 'daemon off;'"
-    ```
+    Docker 17.06 and higher allow you to specify a custom location for a secret
+    within the container. The older version of this command requires you to
+    create a symbolic link to the true location of the `site.conf` file so that
+    Nginx can read it, but the newer version does not require this. The older
+    example is preserved so that you can see the difference.
+
+    - **Docker 17.06 and higher**:
+
+      ```bash
+      $ docker service create \
+           --name nginx \
+           --secret site.key \
+           --secret site.crt \
+           --secret source=site.conf,target=/etc/nginx/conf.d/site.conf \
+           --publish 3000:443 \
+           nginx:latest \
+           sh -c "exec nginx -g 'daemon off;'"
+      ```
 
-    This uses the short syntax for the `--secret` flag, which creates files in
+    - **Docker 17.05 and earlier**:
```

Comment:
Do we have a policy about how far back the docs cover old versions, and when we would remove something like this?